src/Calendar/__stories__/Calendar.stories.js

```diff
@@ -96,6 +98,7 @@ export const dev = () => (
             dateKnobToDate('disable between dates (2)', disabledDateSecondDefault)]}
         locale={text('locale', 'en')}
         openToDate={dateKnobToDate('open to date', new Date())}
+        showToday={boolean('showToday', false)}
```

Comment:
could add compact to as a knob to the dev story as well

Reply:
Added.